Collaboration blog post (#3110)
PostHog/posthog.com: contents/blog/collaborating-on-posthog.md

---
date: 2022-03-24
title: Collaborating on PostHog
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author: ["paolodamico"]
featuredImage: ../images/blog/generic-release-notes.png
featuredImageType: full
categories: ["Product updates"]
---

> PostHog is better accompanied. We quickly realized that as more users join an organization, the incremental value derived by each user increases. Consider for instance when you create a dashboard. You only create it once, but then with each user that joins your organization, the dashboard becomes more valuable. We found clear signals of this in our data: organizations with multiple teammates retain on average 30%+ better. We also recognized that conservatively we could increase insight discovery (our [main Product metric](https://posthog.com/handbook/product/metrics)) by more than 20% by improving collaboration workflows.

Reviewer: > PostHog is better accompanied

Not sure this makes gramatical sense.

> We also recognized that conservatively we could increase insight discovery (our [main Product metric](https://posthog.com/handbook/product/metrics)) by more than 20% by improving collaboration workflows.

We should maybe give a bit more detail on how we estimated this opportunity size.

Author: Added more context.

Armed with the above we set out to make collaboration in PostHog as easily as possible, particularly gearing our efforts to making sure that larger teams are successful, with growing pains that tend to be negligible in small teams. We'll definitely continue improving on this, but we want to show you what we have today.

## Introducing Collaboration

### Use case one: Sharing

- Permalinks
- Granular dashboard permissions

### Use case two: Discovering interesting stuff you weren't looking for
- Project home
- Default dashboard

### Use case three: Leverage what others discovered
- Saved insights + automatic insight naming
- Connect to Data Management

## What's next?

> Still in flux, depends on our learnings, ...

Reviewer: I think we can refer to the roadmap items we have around sharing (from the offsite):
* Slack Previews
* Email subscriptions to dashboards and insights
* Embeddable graph images,

---
date: 2022-03-24

> title: Collaborating on PostHog

Reviewer 2: ```suggestion
title: Introducing Collboration on PostHog
```

Up for debate whether it's 'Collaboration' or 'collaboration' in this context, but I'd like most of these posts to follow headline convention.

rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author: ["paolodamico"]
featuredImage: ../images/blog/generic-release-notes.png
featuredImageType: full
categories: ["Product updates"]
---

> PostHog is better with your team. We quickly realized that as more users join an organization, the incremental value derived by each user increases. Consider for instance when you create a dashboard. You only create it once, but then with each user that joins your organization, the dashboard becomes more valuable. We found clear signals of this in our data: organizations with multiple teammates retain on average 30%+ better. We also recognized that conservatively we could increase insight discovery (our [main Product metric](https://posthog.com/handbook/product/metrics)) by more than 20% by improving collaboration workflows [1]. For instance, we discovered that 50%+ of invited teammates never discover a single insight. If we could get these invited teammates properly onboarded, the overall value derived from PostHog would increase significantly.

Reviewer 2: ```suggestion
PostHog is better with your team. We quickly realized that as more users join an organization, the incremental value derived by each user increases. Consider for instance when you create a dashboard. You only create it once, but then with each user that joins your organization, the dashboard becomes more valuable. 

We found clear signals of this in our data: organizations with multiple teammates retain on average 30%+ better. We also recognized that conservatively we could increase insight discovery (our [main Product metric](https://posthog.com/handbook/product/metrics)) by more than 20% by improving collaboration workflows [1]. For instance, we discovered that 50%+ of invited teammates never discover a single insight. If we could get these invited teammates properly onboarded, the overall value derived from PostHog would increase significantly.
```

Separating into two paras for readability.

> PostHog is better with your team. We quickly realized that as more users join an organization, the incremental value derived by each user increases. Consider for instance when you create a dashboard. You only create it once, but then with each user that joins your organization, the dashboard becomes more valuable. We found clear signals of this in our data: organizations with multiple teammates retain on average 30%+ better. We also recognized that conservatively we could increase insight discovery (our [main Product metric](https://posthog.com/handbook/product/metrics)) by more than 20% by improving collaboration workflows [1]. For instance, we discovered that 50%+ of invited teammates never discover a single insight. If we could get these invited teammates properly onboarded, the overall value derived from PostHog would increase significantly.
> 
> 
> Armed with the above we set out to make collaboration in PostHog as easily as possible, particularly gearing our efforts to making sure that larger teams are successful, with growing pains that tend to be negligible in small teams. We'll definitely continue improving on this, but we want to show you what we have today.

Reviewer 2: ```suggestion
PostHog is better with teammates. You only have to create a dashboard once, but it becomes more valuable every time you invite someone from your organization to view it. And this isn't just some vague truism: it's a measurable fact.

Our data shows organizations with multiple teammates retain 30%+ better than those with a single user. Better collaboration also powers our [main Product metric](https://github.com/handbook/product/metrics): insight discoveries. Our conservative estimates suggest we could increase discoveries by 20% by improving collaboration workflows [1], so that's what we've done.

Over the past few months, we've worked hard to make collaboration in PostHog as easy as possible, particularly for larger teams. That work will never end, but we want to show you what we have today.
```

Have rejigged the opening paras a little to tighten things up.

![Concept of how insight links changed to permalinks today](../images/blog/collaborating-on-posthog_1.png)

Dashboards can easily be shared too, and even if you share a link to a dashboard in a different project, if the user has access to that project, they'll be automatically switched to that project.

For even larger teams, new challenges arise when collaborating with dashboards. In particular, we found a strong need to prevent accidental changes that would affect metrics for others. This was mentioned frequently for larger organizations. In small teams this is usually not problematic as it's common for a single person to own the analytics process. So we introduced granular permission controls to dashboards to provide confidence to users that dashboards haven't changed when they shouldn't.

<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/1_33_0-dashboard-permissions.png" width="500" alt="Screenshot of dashboard permissions" />

### Use case two: Discovering interesting stuff you weren't looking for

> Sometimes you don't know what you're looking for, or you don't know where to start. There's many different avenues to improving your product, and in particular there might be opportunities for improvements across a myriad of places. For instance, we recently [discovered](https://github.com/PostHog/posthog/pull/7973) users seldom use our paths advanced features and these could introduce mental burden to users.

Reviewer 2: ```suggestion
Sometimes you don't know what you're looking for, or you don't know where to start. There are many different avenues to improving your product. For example, we recently [discovered](https://github.com/PostHog/posthog/pull/7973) users seldom use our Paths advanced features and these could introduce mental burden to users.
```

> We created a project homepage with the goal of surfacing helpful insights very quickly that you may not have had in your radar. These insights can hopefully help you find product avenues worth improving. By making it the default page on PostHog, our aim is to help users make quick discoveries. We'll surface your most important product & company metrics here as well as popular insights. We're leveraging social proof from what other teammates are doing in your app to help you discover more product improvements.

Reviewer 2: ```suggestion
To aid serendipitous discoveries, we've created a project homepage to surface helpful insights you may not have considered. By making it the default page on PostHog, our aim is to help users make quick discoveries. We'll surface your most important product and company metrics here as well as popular insights, and we're leveraging social proof from what other teammates are doing in your app to help you discover more product improvements.
```

> In addition, if your team uses the recordings feature we'll also surface the most recent recordings here. We found that on average returning users to the recordings feature discover proportionally more than 3X than other users (more context [here](https://github.com/PostHog/posthog/issues/8595#issuecomment-1056916848)). This should also help drive qualitative discoveries about your product.

Reviewer 2: ```suggestion
In addition, if your team uses the [Session Recording](/product/session-recording), we'll also surface the most recent recordings here. We've found that users who return to session recordings make 3x more discoveries than other users (more context [here](https://github.com/PostHog/posthog/issues/8595#issuecomment-1056916848)). This should also help drive qualitative discoveries about your product.
```

> Mockup / screenshot here.

Reviewer 3: Reminder to add this

Author: @clarkus @rcmarron which mockup or screenshot should we use for the project home page?

Reviewer 4: Defer to @rcmarron but all the designs are at https://www.figma.com/file/gQBj9YnNgD8YW4nBwCVLZf/PostHog-App?node-id=7877%3A87159. The MVP - populated screen is probably the closest to what a user would see. Another idea could be to screenshot this working on our demo project?

> ### Use case three: Leverage what others discovered

Reviewer 2: ```suggestion
## Use case three: Leverage what others discovered
```

> We found two general profiles of users of PostHog, people who are naturally curious and whose job leads them to discover insights all the time and across multiple dimensions, and people who use PostHog to answer specific questions as they arise. For people in the second camp, we wanted to improve the experience so they can leverage what natural discoverers already do. So we made it a lot easier to both save and find saved insights.

Reviewer 2: ```suggestion
We find there are two general profiles for PostHog users:

1. People who are naturally curious and whose job leads them to discover insights all the time and across multiple dimensions.
2. People who use PostHog to answer specific questions as they arise. 
```

> The first part, for people who create insights, we've made it really easy to save now. With one-click saving and automatic insight naming you can be sure to save any useful insight for others to discover. Keep in mind the more detailed title & description, the easier it will be for others to discover useful insights.

Reviewer 2: ```suggestion
For people who create insights, we've introduced one-click saving and automatic insight naming to ensure any insight you create is easy to understand for other users who happen upon them. This means people who create lots of insights but don't have to spend time documentating or explaining them to their colleagues, and their colleagues don't have to waste time decoding what insights mean.
```

![Screenshot of automatic insight naming](../images/blog/collaborating-on-posthog_3.png)

> And for people who love discovering insights that others helped create, we made it easier to search and find saved insights.

Reviewer 2: ```suggestion
And for people who love discovering insights that others helped create, we've made it easier to search and find saved insights.
```

![Screenshot of saved insights](../images/blog/collaborating-on-posthog_2.png)

> Finally worth mentioning that in the context of expanding knowledge sharing and making sure anyone can answer questions with PostHog, we introduced the new [Data Management](link_to_blog_post_here) feature, which aims to bring all the context on how your product is instrumented to your entire organization. No need to ask an engineer or the PM for which event you should use to answer a question.

Reviewer 2: ```suggestion
To support this work, we also introduced the new [Data Management](link_to_blog_post_here) feature, which aims to bring all the context on how your product is instrumented to your entire organization. Now there's no need to ask an engineer or the PM which event you should use to answer a question, which both parties will appreciate!
```

## What's next?
We continue to actively work on improving collaboration throughout the product and we have some specific plans in our short term [roadmap](https://posthog.com/handbook/people/team-structure/team-app#roadmap). However, we're a fast moving company and we continuously reprioritize based on user feedback, market and need changes, and strategy changes, so while our goal to make more people successful with PostHog will remain, the way we tackle this problem and the actual product changes may change.

With the disclaimer above in mind, here's some exciting stuff we hope to ship soon. 
### Slack Previews

> Almost every team uses some sort of IM tool (think Slack, Discord, MS Teams, ...) and in particular, it's a place for discussions. We believe that adding a bunch more context directly in your IM tool when you share a link to a PostHog insight or dashboard will speed up discussions (e.g. a screenshot of the graph, general details, recent changes, ...). We're starting with Slack to test drive this concept, but if successful, expanding to other tools will make sense.

Reviewer 3: ```suggestion
Almost every team uses some sort of Instant Messaging tool (think Slack, Discord, MS Teams, ...) and in particular, it's a place for discussions. We believe that adding a bunch more context directly in your Instant Messaging tool when you share a link to a PostHog insight or dashboard will speed up discussions (e.g. a screenshot of the graph, general details, recent changes, ...). We're starting with Slack to test drive this concept, but if successful, expanding to other tools will make sense.
```

> Almost every team uses some sort of Instant Messaging tool (think Slack, Discord, MS Teams, ...) and in particular, it's a place for discussions. We believe that adding a bunch more context directly in your Instant Messaging tool when you share a link to a PostHog insight or dashboard will speed up discussions (e.g. a screenshot of the graph, general details, recent changes, ...). We're starting with Slack to test drive this concept, but if successful, expanding to other tools will make sense.

Reviewer 2: ```suggestion
Almost every team uses some sort of Instant Messaging tool (think Slack, Discord, Teams, ...) and in particular, it's a place for discussions. We believe that adding more context when you share a link to a PostHog insight or dashboard in your messaging tool will speed up discussions (e.g. a screenshot of the graph, general details, recent changes, ...). We're starting with Slack to test drive this concept, but if successful, expanding to other tools will make sense.
```

 
### Email subscriptions to dashboards and insights

> As we analyzed dashboard usage in PostHog (and from conversations with users too), we discovered is quite common for teams to have 1-2 dashboards that represent critical metrics for their product and their company. These metrics are so important, that keeping an eye out on them continuously is strongly advised (think revenue or number of sales for an e-commerce company, or retention for a pre-PMF startup). A very user friendly way of doing this is integrating with current workflows, and almost everyone uses email. So you'll be able to start your Mondays (or any other day) with a digest of how your product and company are doing.

Reviewer 3: ```suggestion
As we analyzed dashboard usage in PostHog (and from conversations with users too), we discovered it's quite common for teams to have 1-2 dashboards that represent critical metrics for their product and their company. These metrics are so important, that keeping an eye out on them continuously is strongly advised (think revenue or number of sales for an e-commerce company, or retention for a pre-PMF startup). A very user friendly way of doing this is integrating with current workflows, and almost everyone uses email. So you'll be able to start your Mondays (or any other day) with a digest of how your product and company are doing.
```

> As we analyzed dashboard usage in PostHog (and from conversations with users too), we discovered it's quite common for teams to have 1-2 dashboards that represent critical metrics for their product and their company. These metrics are so important, that keeping an eye out on them continuously is strongly advised (think revenue or number of sales for an e-commerce company, or retention for a pre-PMF startup). A very user friendly way of doing this is integrating with current workflows, and almost everyone uses email. So you'll be able to start your Mondays (or any other day) with a digest of how your product and company are doing.

Reviewer 3: ```suggestion
As we analyzed dashboard usage in PostHog (and from conversations with users too), we discovered it's quite common for teams to have 1-2 dashboards that represent critical metrics for their product and their company. These metrics are so important, that keeping an eye out on them continuously is strongly advised (think revenue or number of sales for an e-commerce company, or retention for a pre-Product-Market-Fit startup). A very user friendly way of doing this is integrating with current workflows, and almost everyone uses email. So you'll be able to start your Mondays (or any other day) with a digest of how your product and company are doing.
```

> As we analyzed dashboard usage in PostHog (and from conversations with users too), we discovered it's quite common for teams to have 1-2 dashboards that represent critical metrics for their product and their company. These metrics are so important, that keeping an eye out on them continuously is strongly advised (think revenue or number of sales for an e-commerce company, or retention for a pre-Product-Market-Fit startup). A very user friendly way of doing this is integrating with current workflows, and almost everyone uses email. So you'll be able to start your Mondays (or any other day) with a digest of how your product and company are doing.

Reviewer 2: ```suggestion
As we analyzed dashboard usage in PostHog (and from conversations with users too), we discovered it's quite common for teams to have 1-2 dashboards that represent critical metrics for their product and their company. These metrics, such as number of sales for an e-commerce company or retention for a pre-Product-Market-Fit startup, are so important that keeping an eye on them continuously is vital. A very user friendly way of doing this is integrating with current workflows, and almost everyone uses email. So you'll be able to start your Mondays (or any other day) with a digest of how your product and company are doing.
```

---
date: 2022-03-24
title: Introducing Collboration on PostHog
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author: ["paolodamico"]
featuredImage: ../images/blog/generic-release-notes.png
featuredImageType: full
categories: ["Product updates"]
---

> PostHog is better with teammates. You only have to create a dashboard once, but it becomes more valuable every time you invite someone from your organization to view it. And this isn't just some vague truism: it's a measurable fact.

Reviewer 5: ```suggestion
PostHog is better with teammates. You only have to create a [dashboard](/docs/user-guides/dashboards) once, but it becomes more valuable every time you invite someone from your organization to view it. And this isn't just some vague truism: it's a measurable fact.
```

2. People who use PostHog to answer specific questions as they arise. 

For people who create insights, we've introduced one-click saving and automatic insight naming to ensure any insight you create is easy to understand for other users who happen upon them. This means people who create lots of insights but don't have to spend time documentating or explaining them to their colleagues, and their colleagues don't have to waste time decoding what insights mean.

![Screenshot of automatic insight naming](../images/blog/collaborating-on-posthog_3.png)

And for people who love discovering insights that others helped create, we've made it easier to search and find saved insights.

![Screenshot of saved insights](../images/blog/collaborating-on-posthog_2.png)

To support this work, we also introduced the new [Data Management](link_to_blog_post_here) feature, which aims to bring all the context on how your product is instrumented to your entire organization. Now there's no need to ask an engineer or the PM which event you should use to answer a question, which both parties will appreciate!

## What's next?

> We continue to actively work on improving collaboration throughout the product and we have some specific plans in our short term [roadmap](https://posthog.com/handbook/people/team-structure/team-app#roadmap). However, we're a fast moving company and we continuously reprioritize based on user feedback, market and need changes, and strategy changes, so while our goal to make more people successful with PostHog will remain, the way we tackle this problem and the actual product changes may change.

Reviewer 5: ```suggestion
We're continuing to actively work on improving collaboration throughout the product and we have some specific plans in our short term [roadmap](https://posthog.com/handbook/people/team-structure/team-app#roadmap). 

However, we're a fast moving company and we continuously reprioritize based on feedback, market needs, etc., do the way we tackle this problem may change.
```

Just shortening this a bit

With the disclaimer above in mind, here's some exciting stuff we hope to ship soon. 
### Slack Previews

> Almost every team uses some sort of Instant Messaging tool (think Slack, Discord, Teams, ...) and in particular, it's a place for discussions. We believe that adding more context when you share a link to a PostHog insight or dashboard in your messaging tool will speed up discussions (e.g. a screenshot of the graph, general details, recent changes, ...). We're starting with Slack to test drive this concept, but if successful, expanding to other tools will make sense.

Reviewer 5: ```suggestion
Almost every team uses some sort of Instant Messaging tool (think Slack, Discord, Teams, ...) and in particular, it's a place for discussions. We believe that adding more context when you share a link to a PostHog insight or dashboard will speed up discussions (e.g. a screenshot of the graph, general details, recent changes, ...). We're starting with Slack to test drive this concept, but if successful, expanding to other tools will make sense.
```

 
### Email subscriptions to dashboards and insights

> As we analyzed dashboard usage in PostHog (and from conversations with users too), we discovered it's quite common for teams to have 1-2 dashboards that represent critical metrics for their product and their company. These metrics, such as number of sales for an e-commerce company or retention for a pre-Product-Market-Fit startup, are so important that keeping an eye on them continuously is vital. A very user friendly way of doing this is integrating with current workflows, and almost everyone uses email. So you'll be able to start your Mondays (or any other day) with a digest of how your product and company are doing.

Reviewer 5: ```suggestion
As we analyzed dashboard usage in PostHog, we discovered it's quite common for teams to have 1-2 dashboards that represent their critical metrics, such as number of sales for an e-commerce company or retention for a pre-Product-Market-Fit startup. 

A very user-friendly way of keeping users updated on these metrics is by integrating with their current workflows, such as email. Soon you'll be able to start your Mondays (or any other day) with a digest of how your product and company are doing.
```

This read simpler to me
